datadog/dogshell/dashboard.py

```diff
@@ -81,7 +81,7 @@ def _post(cls, args):
             widgets = sys.stdin.read()
         try:
             widgets = json.loads(widgets)
-        except:
+        except Exception:
```

Comment:
Thanks for this. 
Should we raise specific exceptions there (and below too) as opposed to plain ones?

Reply:
Don't start me on this 😆 

The way this is written and done is indeed pretty bad. I've chosen to ignore it for now as it's not the purpose of this patch, but those exception handlings need serious cleanups indeed.

I'll make a second PR later to clean this up if that's good with you.

Reply:
:). Fair enough, yes I agree it'd need to be a different PR. 

> I'll make a second PR later to clean this up if that's good with you.

Can also look into this if you don't have time. Thanks.